optimize/client/src/components/Reports/Report.js

```diff
@@ -101,11 +101,6 @@ export class Report extends React.Component {
 
   render() {
     const {report, serverError} = this.state;
-    const hasOpensearchError = serverError?.detailedMessage?.includes('No interpreter registered');
-
-    if (hasOpensearchError) {
-      return <ErrorPage text={t('apiErrors.reportNotSupportedForOpenSearch')} />;
-    }
```

Comment:
Hey @grlimacan, 
I assumed that there is always an interpreter to all types of reports and no such error should happen. 
Am i right here?

Reply:
Yes @OmranAbazid , you are right